internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -338,10 +338,25 @@ var rules = []string{
 	"dont_measure fsmagic=0x43415d53", // SMACK_MAGIC
 	"dont_measure fsmagic=0x27e0eb",   // CGROUP_SUPER_MAGIC
 	"dont_measure fsmagic=0x63677270", // CGROUP2_SUPER_MAGIC
-	"dont_measure fsmagic=0x6e736673", // NSFS_MAGIC
+	"dont_measure fsmagic=0x6e736673", // NFS_MAGIC
```

Comment:
This was correct before, its the namespace filesystem.

Reply:
you're right, I have something messed up, fixing the list

Reply:
cool, I thought the kernel docs also had a typo :sweat_smile: